### PR TITLE
Filter refactor

### PR DESCRIPTION
**Pull request by pburnsdata**
# Issues
#84
#91 
#93

### CODE REVIEW

File: src/onDraw/dataExport/deriveData.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default function deriveData() {
             acc[cur] = {
                 site: subjectDatum.sitename,
                 status: subjectDatum.status,
-                freeze: subjectDatum.SubjFreezeFlg
+                freeze: subjectDatum.subjfreezeflg
```
Comment (samussiah) [inline src/onDraw/dataExport/deriveData.js]: use a setting here

File: src/onDraw/flagParentRows.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export default function flagParentRows() {
+    this.rows.classed('grayParent', d => d.filtered & d.visible_child);
```
Comment (samussiah) [inline src/onDraw/flagParentRows.js]: use double `&` here

File: src/onDraw/dataExport/deriveData.js (hunk 1 above)
Comment (pburnsdata) [reply src/onDraw/dataExport/deriveData.js]: all over it.

File: package.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
     "xlsx": "^0.13.0"
   },
   "scripts": {
-    "build": "npm run bundle && npm run format && npm run build-md",
+    "build": "npm run bundle && npm run format && npm run build-md && npm run test-page",
```
Comment (pburnsdata) [inline package.json]: Yeah don't know how we lost this puppy.

**Comment by pburnsdata** [comment]
@samussiah Looks good - moves some of the hard coding out -Nice work. Fixed the double click to collapse problem too.

**Comment by dschwentker** [comment]
Tested and passed #84 #91 and #93 in all browsers.

**Pull request merged**